script/dolphinscheduler-daemon.sh

```diff
@@ -81,8 +79,26 @@ else
   exit 1
 fi
 
+state=""
+function get_server_running_status() {
+  state="STOP"
+  if [ -f $pid ]; then
```

Comment:
Hi @rickchengx, sorry for later reply.  But I still think that using the file method to record the process is unreliable.
Our purpose is just to check whether the master/worker process exists, the current check command is relatively accurate. And this command has contain $DOLPHINSCHEDULER_HOME, so will not check other install path.
```
serverCount=`ps -ef | grep "java" | grep "$DOLPHINSCHEDULER_HOME" | grep "$CLASS" | grep -v "grep" | wc -l`
```
So i think that we just need use this command to check the server status before start the services. If running, no need start again.